mintapi/api.py

```diff
@@ -940,17 +939,16 @@ def mos_to_yrmo(mos_frm_zero):
                     str(max(map(int, response['data']['income'].keys())))
                 ]['bu'],
                 'spend': response['data']['spending'][
-                    str(max(map(int, response['data']['income'].keys())))
+                    str(max(map(int, response['data']['spending'].keys())))
```

Comment:
So you are saying you don't actually notice a bug with the current behavior, or a change in behavior when you change this? That's really confusing since it seems like it should be broken one way or the other...haha.

Reply:
Haha it sure does.  From looking at responses I get back, it seems like the keys in data.income and data.spending _just so happen_ to be the same.  I'm guessing because they're referring to timeframes?  (Months since epoch?)